tools/cockpit.spec

```diff
@@ -374,10 +374,10 @@ The Cockpit component for managing storage.  This package uses Storaged.
 
 %package ostree
 Summary: Cockpit user interface for rpm-ostree
-%if 0%{?rhel}
-Requires: rpm-ostree-client >= 2015.11-1
-%else
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
Can we test for %centos here?  That way we could express the exact requirements for each OS, without having to find a trick that works for all of them.